packages/esm-patient-registration-app/src/constants.ts

```diff
@@ -1,2 +1,8 @@
+import { OmrsOfflineHttpHeaders } from '@openmrs/esm-framework';
+
 export const moduleName = '@openmrs/esm-patient-registration-app';
 export const patientRegistration = 'patient-registration';
+
+export const cacheForOfflineHeaders: OmrsOfflineHttpHeaders = {
+  'x-omrs-offline-caching-strategy': 'network-first',
```

Comment:
One suggestion that I have is that this header name (along the others) might be exported as a constant from `@openmrs/esm-framework`. The benefit would be, that we could change the value of the constant later (making refactoring easier).

Reply:
Good point, that makes a lot of sense. I'll get on it right away!